examples/with-mysql/README.md

```diff
@@ -1,82 +1,102 @@
-# MySQL Example
+# Next.js + MySQL
 
-This is an example of using [MySQL](https://www.mysql.com/) in a Next.js project.
+This is a [Next.js](https://nextjs.org/) project that uses [Prisma](https://www.prisma.io/) to connect to a [PlanetScale](https://planetscale.com/) MySQL database and [Tailwind CSS](https://tailwindcss.com/) for styling.
 
 ## Demo
 
-### [https://next-mysql.vercel.app](https://next-mysql.vercel.app/)
+https://next-mysql.vercel.app
```

Comment:
Looks like the example here has a bunch of spammy entries, should we clean these up?

Reply:
Yep, that's what prompted this - I will redeploy this new code post merge.